### PR TITLE
Fix branding for usage with default profile

### PR DESCRIPTION
**Pull request by andersem**
I thought this was already adressed, but seems not. When using a payment profile like `default` which doesn't contain an account_id, the branding API fails.

Therefore, we've added a new element to the path to specify which account_id we're on.

### CODE REVIEW

File: classes/class-dintero-checkout-widget.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function form( $instance ) {
 		</p>
 		<p>
 			<label for="<?php echo esc_attr( $icon_color ); ?>"><?php esc_html_e( 'Icon color:', 'dintero-checkout-for-woocommerce' ); ?>
-			<input type="color" class="widefat colorpick" value="#cecece" id="<?php echo esc_attr( $icon_color ); ?>" name="<?php echo esc_attr( $icon_color ); ?>"  /></label>	
+			<input type="color" class="widefat colorpick" value="#cecece" id="<?php echo esc_attr( $icon_color ); ?>" name="<?php echo esc_attr( $icon_color ); ?>"  /></label>
```
Comment (andersem) [inline classes/class-dintero-checkout-widget.php:70]: I'm not sure what these changes are. Maybe IntelliJ auto-fixing some syntax.

Comment (montazarr) [reply classes/class-dintero-checkout-widget.php:70]: I've made some adjustments to the formatting.

**Comment by andersem** [comment]
> 

I think you need to recreate the widgets

**Comment by montazarr** [comment]
The issue seem to be that the URL refers to a non-existing resource. This doesn't concern the WC Widget itself, however, since it uses the same URL, it is also effected by this issue.

**Review by andersem — COMMENTED** [review]
I think you can remove the `environment` function

File: classes/class-dintero-checkout-widget.php [hunk 2]
```diff
@@ -99,6 +99,10 @@ public function update( $new_instance, $old_instance ) {
 		return $instance;
 	}
 
+	private function environment( $settings ) {
```
[the remark this answers is not in the document]
Comment (montazarr) [reply classes/class-dintero-checkout-widget.php:102]: I forgot to remove it. It is no longer required.

**Pull request merged**